### PR TITLE
[IMP] quality_control_mrp: take product from manufacturing order + extended search view (*_stock)

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

File: quality_control_mrp/models/qc_inspection.py [hunk 1]
```diff
@@ -8,15 +8,24 @@
...
     @api.depends('object_id')
     def get_production(self):
-        self.production = False
-        if self.object_id:
-            if self.object_id._name == 'stock.move':
-                self.production = self.object_id.production_id
-            elif self.object_id._name == 'mrp.production':
-                self.production = self.object_id
+        for inspection in self:
+            inspection.production = False
```
Comment (pedrobaeza) [inline quality_control_mrp/models/qc_inspection.py]: This is not needed anymore

**Comment by oihane** [comment]
Travis errors are not related to the module in this PR.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by CarlosMolinaP** [comment]
LGTM :+1:

**Comment by Daniel-CA** [comment]
:+1:

**Pull request merged**